recipes/vscode-python/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "vscode-python" %}
+{% set version = "2021.2.582707922" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://github.com/microsoft/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 517360d2af77dc7376d02cdfdf9348142d1d2c7b792e5ff6c548871b2aa4e991
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - nodejs ==12.14.1
+    - code-server >=3.9
+  host:
+    - nodejs ==12.14.1
+    - code-server >=3.9
+  run:
+    - code-server >=3.9
+    - Python >=2.7
+    - click ==7.1.2
+    - isort ==5.7.0
+    - jedi-language-server ==0.21.0
+    - jedi ==0.17.2
+    - parso ==0.7.1
+    - pygls ==0.9.1
```

Comment:
These pins look way too strict, is this really necessary?

Reply:
I've just copied the pins from the [requirements.txt](https://github.com/microsoft/vscode-python/blob/main/requirements.txt). I replaced them with the pins from the [requirements.in](https://github.com/microsoft/vscode-python/blob/main/requirements.in)